pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[tool.poetry]
+name = "peewee-model-utils"
+version = "0.1.0"
+description = ""
+authors = ["Rahul Mondal <rahul.mondal@ingrammicro.com>"]
+readme = "README.md"
+packages = [{include = "connect"}]
+
+[tool.poetry.dependencies]
+python = ">=3.7,<4"
+peewee = "3.15.4"
+
+[tool.poetry.group.test.dependencies]
+pytest = "^6.1.2"
+pytest-cov = "^2.10.1"
+pytest-mock = "^3.3.1"
+coverage = {extras = ["toml"], version = "^5.3"}
+flake8 = "~3.8"
+flake8-bugbear = "~20"
+flake8-cognitive-complexity = "^0.1"
+flake8-commas = "~2.0"
+flake8-future-import = "~0.4"
+flake8-import-order = "~0.18"
+flake8-broken-line = "~0.3"
+Sphinx = "^4.5.0"
```

Comment:
Dependency on sphinx is not required since you don't have rst documentation. Moreover for documentation we moved to mkdocs with material-mkdocs plugin. 
If you decide to add documentation for read the docs take a look at https://github.com/cloudblue/connect-eaas-core.